### PR TITLE
Missing quotes surrounding "foo"

### PR DESCRIPTION
**Pull request by chaoszcat**
If no quotes, this sample code will only print "I like and bar"

### CODE REVIEW

**Comment by ninsuo** [comment]
I would add `{% set foo = "foo" %}` instead, just to show that we can use variables with `replace`

**Comment by chaoszcat** [comment]
@ninsuo nice. That would be good too. But this patch is just a direct fix on what's not working :)

File: doc/filters/replace.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ The ``replace`` filter formats a given string by replacing the placeholders
 
 .. code-block:: jinja
 
-    {{ "I like %this% and %that%."|replace({'%this%': foo, '%that%': "bar"}) }}
+    {{ "I like %this% and %that%."|replace({'%this%': "foo", '%that%': "bar"}) }}
```
Comment (fabpot) [inline doc/filters/replace.rst]: That's wrong. We do want a var here (see the explanation just below)

Comment (chaoszcat) [inline doc/filters/replace.rst]: Oops sorry didn't notice the purple comment below

**Pull request closed without merging**